### PR TITLE
Wavedrom updates - machine.adoc

### PR DESCRIPTION
**Pull request by kersten1**
Fixing up wavedrom things one chapter at a time. Weirdly started with machine.adoc. Just roll with it. 

- Removed color from files. Changes indicated from:
https://github.com/riscv/riscv-isa-manual/issues/1430

- Moved inline wavedrom files to be separate files and then used include to pull them in.

### CODE REVIEW

File: src/images/wavedrom/trap-return.adoc [hunk 1]
```diff
@@ -4,10 +4,10 @@
...
-  {bits: 5, name: 'rd', type: 2, attr: ['5','0'],},
-  {bits: 3, name: 'funct3', type: 8, attr: ['3','PRIV'],},
-  {bits: 5, name: 'rs1', type: 4, attr: ['5','0'],},
-  {bits: 12, name: 'funct12', type: 8, attr: ['12','MRET/SRET',]},
+  {bits: 7, name: 'opcode'},
+  {bits: 5, name: 'rd'},
+  {bits: 3, name: 'funct3'},
+  {bits: 5, name: 'rs1'},
+  {bits: 12, name: 'funct12'},
 ], config: {bits: 32}}
```
[the remark this answers is not in the document]
Comment (kersten1) [reply src/images/wavedrom/trap-return.adoc:12]: Thank you! added them back in.

**Review by aswaterman — APPROVED** [review]
I didn't perform a detailed code review, but I certainly approve in principle!

**Comment by aswaterman** [comment]
@kersten1 this PR seems to have broken the build:

```
asciidoctor: ERROR: images/bytefield/mideleg.adoc: line 2: Failed to generate image: undefined method `[]' for nil:NilClass
asciidoctor: ERROR: images/bytefield/sie.edn: line 2: Failed to generate image: undefined method `[]' for nil:NilClass
asciidoctor: ERROR: hypervisor.adoc: line 587: Failed to generate image: unexpected token at ''
```

**Comment by kersten1** [comment]
@aswaterman I can't imagine why as I didn't touch those files at all. Looking.... Although I'm not the best at debugging builds yet....

**Comment by aswaterman** [comment]
It looks like there is a concurrency problem with parallel builds; `make -j1` works but `make -j8` fails.  Most likely, the addition of more wavedrom figures increased the probability of failure.  So, not your fault.  I'll see if I can figure out what to do about this.

**Comment by aswaterman** [comment]
OK, it looks like the issue is that the wavedrom builds are creating temporary files inside the source tree, and so there is a race condition between multiple builds.  The easy fix is to create a temporary copy of the source tree for each individual build.  I'll handle it.

**Comment by aswaterman** [comment]
BTW @kersten1 sorry for the false alarm about this PR; you were just unlucky enough to step on this pre-existing land mine :)

**Comment by kersten1** [comment]
@aswaterman I just felt bad that I was useless at debugging.... Bill is back tomorrow and can help too.

**Comment by aswaterman** [comment]
No prob.  Fixed in https://github.com/riscv/riscv-isa-manual/pull/1491

**Pull request merged**